### PR TITLE
Add source file

### PR DESCRIPTION
**Pull request by zanderz**
This will replace the .vbs files when installing and at startup. A simpler version of utilities such as: http://www.ntwind.com/software/hstart.html
@maxtaco @gabriel

### CODE REVIEW

File: kbrunquiet.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	
+	attr := &syscall.ProcAttr{
+		Files: []uintptr{uintptr(syscall.Stdin),uintptr(syscall.Stdout),uintptr(syscall.Stderr)},
+		Env: syscall.Environ(),
+		Sys: &syscall.SysProcAttr{
+			HideWindow: true,
+			CreationFlags: CREATE_NEW_CONSOLE,	
+		},
+	}
+	fmt.Printf("Launching %s with args %v\n", os.Args[argsIndex], os.Args[argsIndex:])
```
Comment (maxtaco) [inline kbrunquiet.go]: where do these log lines wind up?  Do we want them in production?

**Comment by maxtaco** [comment]
Why have this as a standalone repo rather than being part of the main app?

**Comment by zanderz** [comment]
I got scared reading stuff like this:
http://stackoverflow.com/questions/15952892/using-the-console-in-a-gui-app-in-windows-only-if-its-run-from-a-console?lq=1
An executable has to be marked as one or the other in Windows (console or windowing). If it is marked windows and tries to open a console after launching, there can be races getting the right stdin stream, or the invoking console not waiting for it to finish, etc. If it is marked console and switches to windows, we're back to the ugly flashing console window at launch, which was the original reason we went down this particular rabbit hole. The fact that programs like Visual Studio and Java use the 2-executable solution leads me to believe there probably isn't anything better.

**Comment by maxtaco** [comment]
Ok, I kind of understand, but could it live as a parallel top-level directory in `keybase/client`?

File: kbrunquiet.go (hunk 1 above)
Comment (zanderz) [inline kbrunquiet.go]: In production, this program is only used at startup and at install time, and nobody will see these lines. You can see them if you invoke the program manually at the command line, but it just echoes the command you just typed in. Command line arguments for every running process in the system are visible anyway.

File: kbrunquiet.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	"strings"
+	"time"
+)
+
+const CREATE_NEW_CONSOLE = 0x00000010
+
+func main() {
+	argsIndex := 1	// 0 is the name of this program, 1 is either the one to launch or the "wait" option
+	
+	if (len(os.Args) < 2){
```
Comment (gabriel) [inline kbrunquiet.go]: why parens around if...

are you running go fmt?

File: kbrunquiet.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+import (
+	"syscall"
+	"os"
+	"fmt"
+	"strings"
+	"time"
+)
+
+const CREATE_NEW_CONSOLE = 0x00000010
```
Comment (gabriel) [inline kbrunquiet.go]: I don't think go consts are of this format..

maybe use the go linter, which should warn you of this type of stuff

File: kbrunquiet.go [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+			CreationFlags: CREATE_NEW_CONSOLE,	
+		},
+	}
+	fmt.Printf("Launching %s with args %v\n", os.Args[argsIndex], os.Args[argsIndex:])
+	pid, handle, err := syscall.StartProcess(os.Args[argsIndex], os.Args[argsIndex:], attr )
+	fmt.Printf("%v, %v, %v\n", pid, handle, err)
+	if doWait {
+		p, err := os.FindProcess(pid)
+		if err != nil {
+			fmt.Printf("Launcher can't find %d\n", pid)
```
Comment (gabriel) [inline kbrunquiet.go]: Good to include error when printing..
I would use log.Printf instead of fmt.Printf

**Pull request closed without merging**